common/on_actions/imperia_on_monthtly_pulse.txt

```diff
@@ -327,13 +327,27 @@ on_imperia_monthly_pulse_ai_congress_voting = {
 					NOT = {
 						has_global_variable = imperia_vienna_congress_minimum_time
 					}
+					imperia_vienna_congress_in_session = yes
 				}
 				# Have a 20% chance of ending it
 				random = {
 					chance = 20
 					end_congress_scripted_effect = yes
 				}
 			}
+			if = {
+				limit = {
+					not = {
+						exists = global_var:imperia_vienna_congress_session_initiator
+					}
+				}
+				remove_global_variable = imperia_vienna_congress_in_session
+				remove_global_variable = congress_random_events_enabled
+				set_variable = {
+					name = 	imperia_vienna_congress_in_session
+					value = no
+				}
```

Comment:
Shouldn't this also be up at the random chance?

Reply:
this is not ai this just ends the congress when there is no host anymore

Reply:
Oh right, `end_congress_scripted_effect` already does that :p